Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,50 @@
 ﻿using System;
+using System.Diagnostics;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    public static class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                string name;
+                Console.Write("Please, tell me your name: ");
+                name = Console.ReadLine();
+
+                
+                string surname;
+                Console.Write("Please, tell me your surname: ");
+                surname = Console.ReadLine();
+
+                
+                int age = 0;
+                Console.Write("Please, tell me your age: ");
+                age = Convert.ToInt32(Console.ReadLine());
+
+                
+                double weight = 0;
+                Console.Write("Please, tell me your weight: ");
+                weight = Convert.ToDouble(Console.ReadLine());
+
+                
+                double height = 0;
+                Console.Write("Please, tell me your height (in cm): ");
+                height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine("\n{0} {1} is {2} years old, his weight is {3} kg and his height is {4} cm.", name, surname, age, weight, height);
+
+                double BMI = Math.Round(CalculateBMI(weight, height), 2);
+
+                Console.WriteLine("His BMI is: {0} \n", BMI);
+            }
+
+        }
+        static double CalculateBMI(double weight, double height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4828859)

Reply:
Done